test/transcoding/SpecConstantComposite.ll

```diff
@@ -0,0 +1,135 @@
+; RUN: llvm-as < %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o %t.spt
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv %t.spv -r -o - | llvm-dis -o %t.ll
```

Comment:
I suggest adding a separate test to check `--spec-const` and `--spec-const-info` command line options with composite spec constants

Reply:
Good idea, thanks.

Reply:
I have added the test in 994ea05.
`--spec-const-info` doesn't show information about composite specialization constants. Intention of this option was to provide information required for specialization via OpenCL API for example. But, since, we can't specialize a composite directly there was no point to add handling for OpSpecConstantComposite. Please let me know if you think we should extend `--spec-const-info` to show composite specialization constants (that may require changing or adding new API to LLVMSPIRVLib).

Reply:
I think there is no need to print info about composite spec constants since we can't set them through OpenCL API, but I wanted to check that this functionality won't break in presence of composite spec constant, i.e. a simple test that all elements of a composite are correctly printed by this option

Reply:
I'm confused a bit. @AlexeySachkov the test added in 994ea05 - is it enough?

Reply:
> @AlexeySachkov the test added in 994ea05 - is it enough?

Yes